geometries/vel_sc_los/vel_sc_los_algorithm.py

```diff
@@ -1,44 +1,34 @@
 import spiceypy as sp
 
-def vel_sc_los(utc, target, galaxy_targ, instr):
-    """S/C LOS velocity wrt boresight direction due to S/C orbit around Earth. Should be positive
-    when S/C and target are receding from each other, and negative otherwise.
+def vel_sc_los(utc, instr):
+    """S/C LOS velocity wrt boresight direction due to S/C orbit around
+    Earth. Should be positive when S/C and target are receding from each
+    other, and negative otherwise.
 
     Args:
         utc (str): date and time at which S/C LOS velocity will be found
-        target (str): S/C name
-        galaxy_targ (str): body ID for galaxy contained in mkfile
-        instr (str): slit name of S/C
+        instr (str): instrument name
 
     Returns:
-        tuple: S/C LOS velocity
+        float: S/C LOS velocity
     """
 
-    # # # # # PART 1: UNIT VECTOR (DIRECTION) OF BORESIGHT # # # # #
+    # # # # # PART o: INSTRUMENT FRAME AND BORESIGHT VECTOR, AND OBSERVER THAT HAS INSTRUMENT # # # # #
 
-    et = sp.utc2et(utc)
-    ref = 'J2000'
-    abcorr = 'NONE'
-    instr_id = sp.bodn2c(instr)
-    galaxy_targ_id = int(galaxy_targ)
-    target_id = sp.bodn2c(target)
+    ref, bsight = sp.getfov(sp.bodn2c(instr),99,99,99)[1:3]
+    obs = sp.bodc2s(sp.frinfo(sp.namfrm(ref))[0])  #btc frame center
```

Comment:
Again, use instrument frame for vectors, and get observer from instrument frame info